granule_ingester/pyproject.toml

```diff
@@ -0,0 +1,59 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+[tool.poetry]
+name = "sdap_granule_ingester"
+version = "1.2.0"
+description = "Ingester component for Apache SDAP"
+authors = ["SDAP PPMC <dev@sdap.apache.org>"]
+license = "Apache-2.0"
+readme = "README.md"
+packages = [
+    { include = "granule_ingester" }
+]
+repository = "https://github.com/apache/incubator-sdap-ingester"
+classifiers=[
+    'Development Status :: 6 - Mature',
+    'Intended Audience :: Science/Research',
+    'Operating System :: OS Independent',
+    'Programming Language :: Python :: 3.8',
+    "License :: OSI Approved :: Apache Software License"
+]
+
+[tool.poetry.dependencies]
+python = ">=3.8.17,<3.11"
+cassandra-driver = "3.23.0"
+aiomultiprocess = "0.7.0"
+aioboto3 = "8.3.0"
+tblib = "1.6.0"
+pysolr = "3.9.0"
+kazoo = "2.8.0"
+aio-pika = "6.7.1"
+elasticsearch = { version = "8.11.0", extras = ["async"]}
+urllib3 = "1.26.2"
+numpy = "1.21.6"
+scipy = "1.7.3"
+netcdf4 = "1.5.3"
+pandas = "1.3.5"
+pytz = "2019.3"
+xarray = "*"
+pyyaml = "5.3.1"
+aiohttp = ">=3.8.0"
+tenacity = "8.2.3"
+requests = ">=2.27.1"
+
```

Comment:
No dev dependencies?

Reply:
I don't think GI uses anything additional for testing, etc